docs/README.md

```diff
@@ -1,8 +1,10 @@
-# differentialabundance/differentialabundance: Documentation
+# nf-core/differentialabundance: Documentation
 
-The differentialabundance/differentialabundance documentation is split into the following pages:
+The nf-core/differentialabundance documentation is split into the following pages:
 
 - [Usage](usage.md)
   - An overview of how the pipeline works, how to run it and a description of all of the different command-line flags.
 - [Output](output.md)
   - An overview of the different results produced by the pipeline and how to interpret them.
+
+You can find a lot more documentation about installing, configuring and running nf-core pipelines on the website: [https://nf-co.re](https://nf-co.re)
```

Comment:
```suggestion
You can find a lot more documentation about installing, configuring and running nf-core pipelines on the website: [https://nf-co.re/docs](https://nf-co.re/docs)
```

Reply:
This change leads to errors like `files_unchanged: docs/README.md does not match the template `